docs/samples/v1beta1/torchserve/README.md

```diff
@@ -0,0 +1,134 @@
+# Predict on a InferenceService using Torchserve
+
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/#install-kfserving).
+2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
+
+## Create PV and PVC
+This document uses amazonEBS PV 
+
+### Create PV
+
+Edit volume id in pv.yaml file
+
+```
+kubectl apply -f pv.yaml
+```
+
+Expected Output
+
+```
+persistentvolume/model-pv-volume created
+```
+
+### Create PVC
+
+```
+kubectl apply -f pvc.yaml
+```
+
+Expected Output
+
+```
+persistentvolumeclaim/model-pv-claim created
+```
+
+### Create PV Pod
+
+```
+kubectl apply -f pvpod.yaml
+```
+
+Expected Output
+
+```
+pod/model-store-pod created
+```
+
+### Copy mar file and config properties to storage
+
+Download Marfile
+
+```
+wget https://torchserve.pytorch.org/mar_files/mnist.mar
+```
+
+Download input image
+
+```
+wget https://raw.githubusercontent.com/pytorch/serve/master/examples/image_classifier/mnist/test_data/0.png
+```
+
+Copy Marfile
+
+```
+kubectl exec --tty pod/model-store-pod -- mkdir /pv/model-store/
+kubectl cp mnist.mar model-store-pod:/pv/model-store/mnist.mar
+```
+
+Copy config.properties
+
+```
+kubectl exec --tty pod/model-store-pod -- mkdir /pv/config/
+kubectl cp config.properties model-store-pod:/pv/config/config.properties
+```
+
+### Delete pv pod
+
+Since amazon EBS provide only ReadWriteOnce mode
+
+```
+kubectl delete pod model-store-pod -n kfserving-test
+```
+
+## Create the InferenceService
+
+Apply the CRD
+
+```
+kubectl apply -f torchserve.yaml
+```
+
+Expected Output
+
+```
+$ inferenceservice.serving.kubeflow.org/torchserve created
+```
+
+## Run a prediction
+The first step is to [determine the ingress IP and ports](../../../README.md#determine-the-ingress-ip-and-ports) and set `INGRESS_HOST` and `INGRESS_PORT`
+
+
+```
+MODEL_NAME=torchserve
+SERVICE_HOSTNAME=$(kubectl get route torchserve-predictor-default -o jsonpath='{.status.url}' | cut -d "/" -f 3)
```

Comment:
I think you can do the top level virtual service `torchserve` as well

Reply:
Will change it to istio virtual service once our KFServing envelope is in done.